.github/workflows/extract-translation-source-files.yml

```diff
@@ -105,9 +108,93 @@ jobs:
           # push changes to branch
           git push
 
-  merge-translations:
+  js-translations:
+    strategy:
+      # using max-parallel to avoid git push/pull issues when running in parallel
+      max-parallel: 1
+      matrix:
+        # repos missing extract_translations target in makefile:
+        # * frontend-app-admin-portal
+        # * frontend-app-publisher
+        # * frontend-platform
+        # * frontend-enterprise?
+        # * frontend-app-learner-portal-enterprise
+        # * frontend-build?
+        # * frontend-app-learner-portal-programs
+        # * frontend-component-cookie-policy-banner
+        # * frontend-app-programs-dashboard
+        # studio-frontend has a different file structure
+        # repos with errors running extract_translations
+        # * frontend-template-application
+        repo:
+          - frontend-app-payment
+          - frontend-app-discussions
+          - frontend-app-enterprise-public-catalog
+          - frontend-app-learning
+          - frontend-app-authn
+          - frontend-app-course-authoring
+          - frontend-app-learner-record
+          - frontend-app-support-tools
+          - frontend-component-header
+          - frontend-component-footer
+          - frontend-app-library-authoring
+          - frontend-app-account
+          - frontend-app-profile
+          - frontend-app-ecommerce
+          - frontend-app-gradebook
+          - frontend-app-program-console
```

Comment:
Mind adding frontend-app-ora-grading as well?  Reason: it's being explicitly considered [for Olive](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3571056752/Olive+MFE+Candidate+Requirements+Matrix).

Reply:
ah, it's on the `edx` org, not the `openedx` org, that's why i missed it https://github.com/edx/frontend-app-ora-grading